Dialogue/Afin/np_010030.csv

```diff
@@ -19,30 +19,30 @@ Afin_favs_present_080#0,"""Received \""Homing Emission\"" from Afin."""
+11_voice_NPC_5003_108#0,"""Since you're still inexperienced, the amount of planets you can go are limited. """
+Afin_04_020#0,"""If you want to go to more planets then work hard to be recognized. The road ahead is a long one."""
+11_voice_NPC_5003_104#0,"""Even though ARCs have a high standing, that doesn't mean they can do whatever they want. """
+Afin_05_020#0,"""There have been times when some outrageous actions have been overlooked, unofficially."""
+11_voice_NPC_5003_100#1,"""It's so nice here. I want to relax all the time. But as soon as there is a mission I have to leave...."""
+11_voice_NPC_5003_100#2,"""There are so many skills it's hard to remember them all isn't it? """
+Afin_07_020#0,"""But, if you can remember them then they're really useful. Let's keep it up!"""
+11_voice_NPC_5003_104#1,"""Hmmm. I don't have any good weapons. I wonder if there is some new ones I could get..."""
+11_voice_NPC_5003_105#0,"""The goal is to find people, so I travel light!"""
 Afin_09_020#0,"""……って言ってたら<br>真面目に戦えって怒られちまった。<br>不真面目なつもりはないのになー。"""
 11_voice_NPC_5003_105#1,"""おれたちはクラスを変更できるけど<br>先輩方は適性で決まっちゃうんだよな。<br>……なんで変更可能になったんだろ？"""
-11_voice_NPC_5003_104#2,"""アークスのなかには、戦闘だけじゃなく<br>情報屋的なチームもいるらしいぜ。<br>……おれもそうなろうかなぁ。"""
-11_voice_NPC_5003_104#3,"""うーん、人探しってむずいもんだな<br>わかっちゃいたけど時間かかりそうだ。<br>ま、焦らずゆっくりやるかな。"""
-11_voice_NPC_5003_109#0,"""最近、ふいにダーカー出現！<br>みたいなのが多い気がするんだよな。<br>おれの気のせいかな？"""
-11_voice_NPC_5003_101#2,"""クライアントオーダーこなして<br>いろいろ感謝されるとさ、なんだか<br>こっちも嬉しくなるよな。へへっ。"""
-11_voice_NPC_5003_108#1,"""お前は勇気がないけど、勇気がないのを<br>自覚してるからいいって、この前先輩に<br>言われたんだよ。失礼しちゃうよなぁ。"""
-Afin_15_020#0,"""でも、一番怖いのは、勇気がないのに<br>あると勘違いしてるヤツなんだってさ。<br>……いわゆる、無謀ってやつなのかな。"""
-11_voice_NPC_5003_105#2,"""ふと思ったんだけどさ、ダーカーって<br>いつからおれたちの敵なんだろうな？<br>いやあ、歴史には疎くて……"""
-11_voice_NPC_5003_104#4,"""アークスは皆の憧れ……だけども<br>こんなにしんどいもんだとは<br>思わなかったなぁ……はあ。"""
-11_voice_NPC_5003_100#3,"""アークス内で一番えらい人たち……<br>六芒均衡だっけ？\u3000どんなんなんだろ。<br>揃いも揃ってムキムキだったり？"""
+11_voice_NPC_5003_104#2,"""There are some ARCs that are not only good at fighting but gathering info. I wonder if I'm the same..."""
+11_voice_NPC_5003_104#3,"""I knew finding people would be hard but this will take a while. Guess I'll just have to be patient."""
+11_voice_NPC_5003_109#0,"""Another Darker appeared! At least that's what it feels like a lot these days. Maybe it's just me?"""
+11_voice_NPC_5003_101#2,"""Seeing you complete so many client orders is making me happy too. Hehe."""
+11_voice_NPC_5003_108#1,"""I was told by my superiors that “you don't have any courage but you know that”. A bit harsh don't you think?"""
+Afin_15_020#0,"""The thing that's scary is that there are people who think they have courage when they don't. ...Guess they're just reckless."""
+11_voice_NPC_5003_105#2,"""I've been thinking. The Darker have been our enemies for a while now but It hasn't been properly been recorded in history..."""
+11_voice_NPC_5003_104#4,"""ARCs are...loved by everyone. But I never thought it would be this tiring...ha-."""
```

Comment:
first off, it's ARKS not ARCs, please see https://github.com/Arks-Layer/PSO2ENPatchCSV/wiki/Story

Reply:
I realized that a short while after submitting it. Thank you for the link, I shall bookmark it.